src/environmentd/src/http/sql.rs

```diff
@@ -116,31 +227,198 @@ impl SqlResult {
         }
     }
 
-    fn ok(client: &mut SessionClient, res: ExecuteResponse) -> SqlResult {
+    fn ok(client: &mut SessionClient, tag: Option<String>) -> SqlResult {
```

Comment:
This is subjective, but I think it's more clear for this take take `tag: String` and have the caller call `expect`. That way it's much more clear that this function is being called in the context of a tag-generating result. Also it makes it harder for someone to misuse in the future who doesn't realize it will panic if `tag` is `None`.

Reply:
Agree. Done.